src/h5web/BreadcrumbsBar.tsx

```diff
@@ -48,26 +48,17 @@ function BreadcrumbsBar(props: Props): JSX.Element {
           </span>
         </h1>
       )}
-      <div role="tablist" className={styles.modeToggler}>
-        <button
-          type="button"
-          role="tab"
-          className={styles.btn}
-          aria-selected={!isInspecting}
-          onClick={() => onSetInspecting(false)}
-        >
-          Display
-        </button>
-        <button
-          type="button"
-          role="tab"
-          className={styles.btn}
-          aria-selected={isInspecting}
-          onClick={() => onSetInspecting(true)}
-        >
-          Inspect
-        </button>
-      </div>
+      <ToggleGroup
+        role="tablist"
+        ariaLabel="Viewer mode"
+        value={String(isInspecting)}
+        onChange={val => {
+          onChangeInspecting(val === 'true' || false);
+        }}
+      >
+        <ToggleGroup.Btn label="Display" value="false" />
+        <ToggleGroup.Btn label="Inspect" value="true" />
```

Comment:
`ToggleGroup.Btn` is very similar to `ToggleBtn`, except that it reads the current `string` value of its parent toggle group from context (instead of reading its own `boolean` value from a prop). I will probably move their styles into a shared global utility class later on.